_includes/partials/homePage-integrations.html

```diff
@@ -1,83 +1,166 @@
+<style>
```

Comment:
@shubhusion why the existing search bar that is in catalog page is being not reused? Why we have to create new component and styles? Please reuse that

Reply:
@shubhusion i didn't heard from you...

Reply:
I will make the necessary changes, but I'm a bit busy this week. Please give me some time.

Reply:
@shubhusion 

> @shubhusion why the existing search bar that is in catalog page is being not reused? Why we have to create new component and styles? Please reuse that

The styles from the existing search bar on the catalog page were duplicated and adapted for the new search bar on the home page. Some modifications were made to ensure it fits well on the home page.

To reuse the same styles, I will need to create a shared .css file that is accessible to both pages.